src/graphql/root/mutation/ln-invoice-payment-send.ts

```diff
@@ -18,31 +23,39 @@ const LnInvoicePaymentSendMutation = GT.Field({
   args: {
     input: { type: GT.NonNull(LnInvoicePaymentInput) },
   },
-  resolve: async (_, args, { user, wallet, logger }) => {
-    const { paymentRequest, memo } = args.input
+  resolve: async (_, args, { ip, domainUser, wallet, logger }) =>
+    addAttributesToCurrentSpanAndPropagate(
+      {
+        [SemanticAttributes.ENDUSER_ID]: domainUser?.id,
+        [ENDUSER_ALIAS]: domainUser?.username,
+        [SemanticAttributes.HTTP_CLIENT_IP]: ip,
+      },
```

Comment:
can't be added in context?

Reply:
This is not a in the call stack of context - so no it cannot.

You can only influence things that are in the direct call graph.

It would only work if :
```
context() {
  resolver()
}
```

But actually its:
```
someOuterFunction() {
  context() // cannot influence attributes set in resolver()
  resolver()
}
```